sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCV2Suite.scala

```diff
@@ -221,4 +221,36 @@ class JDBCV2Suite extends QueryTest with SharedSparkSession {
       checkAnswer(sql("SELECT name, id FROM h2.test.abc"), Row("bob", 4))
     }
   }
+
+  test("DataFrameWriter: jdbc") {
```

Comment:
Other tests above have in their title `DataFrameWriterV2` but this one `DataFrameWriter`. Do you suppose to test v1 mostly here?

Reply:
Other tests have titles `DataFrameWriterV2` because these are testing `DataFrameWriterV2` APIs such as `DataFrameWriterV2.create` or `DataFrameWriterV2.replace`. `jdbc` is a `DataFrameWriter` API.